### PR TITLE
[graphql] add comments and fix examples

### PR DESCRIPTION
**Pull request by stefan-mysten**
## Description 

Fix the examples we have. 

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-graphql-rpc/examples/balance_connection/balance_connection.graphql [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     balance(type:"0xc060006111016b8a020ad5b33834984a437aaa7d3c74c18e09a95d48aceab08c::coin::COIN") {
       coinObjectCount totalBalance
     }
     balanceConnection {
       nodes {
+        coinType
         coinObjectCount
         totalBalance
       }
-      pageInfo {endCursor}
```
Comment (oxade) [inline crates/sui-graphql-rpc/examples/balance_connection/balance_connection.graphql:11]: why take this out?

**Comment by amnn** [comment]
Regarding formatting -- let's not do this by hand, it's a waste of our time. `prettier` is a JS/TS formatter that we use, and it supports formatting GraphQL too.  It can format all our examples with:

```
prettier --no-config --write crates/sui-graphql-rpc/examples/**/*.graphql
```

**Comment by stefan-mysten** [comment]
Agreed with formatting. I went through all examples and tried them out, and sometimes changed some things.

**Comment by stefan-mysten** [comment]
> Regarding formatting -- let's not do this by hand, it's a waste of our time. `prettier` is a JS/TS formatter that we use, and it supports formatting GraphQL too. It can format all our examples with:
> 
> ```
> prettier --no-config --write crates/sui-graphql-rpc/examples/**/*.graphql
> ```

Just ran prettier now so they're all consistent.

File: crates/sui-graphql-rpc/examples/balance_connection/balance_connection.graphql (hunk 1 above)
Comment (stefan-mysten) [reply crates/sui-graphql-rpc/examples/balance_connection/balance_connection.graphql:11]: Fixed

**Review by amnn — APPROVED** [review]
Looks great!

**Review by oxade — APPROVED** [review]
lgtm

**Pull request merged**